lms/envs/devstack.py

```diff
@@ -100,41 +100,6 @@ def should_show_debug_toolbar(_):
     'debug_toolbar_mongo.panel.MongoDebugPanel',
 )
 
-################################ DEBUG TOOLBAR ################################
```

Comment:
They're actually further up in this very file. The conflict resolution initially made it have both at once, so I had to go back and clean that out.